modules/acs-requirements.adoc

```diff
@@ -42,7 +42,9 @@ Starting with {product-title-short} 4.3, both Central and secured cluster servic
 See the default memory and CPU requirements for each component and ensure that the node size can support them.
 ====
 
-Persistent storage by using persistent volume claim (PVC):
+Persistent storage by using persistent volume claim (PVC). A PVC is required on the the cluster where Central is installed. It is strongly recommended on the secured clusters where Scanner V4 is enabled.
+
+Follow these guidelines for PVCs:
 
 * Use Solid-State Drives (SSDs) for best performance. However, you can use another storage type if you do not have SSDs available.
```

Comment:
Just referring to what I have seen being discussed on Slack: Isn't SSD not only "nice to have" (for good performance), but according to the latest findings also required for a stable functioning of Scanner V4, @RTann?